src/context/Search/suggestions/match/rankMatches.test.ts

```diff
@@ -0,0 +1,85 @@
+import { rankMatches } from "./rankMatches";
+
+describe("rankMatches", () => {
+  it.each([
+    [
+      "mathematics",
+      [
+        { slug: "computing", matched: "cs" },
+        { slug: "maths", matched: "mathematics" },
+      ],
+      "maths",
+    ],
+    [
+      "year 10",
+      [
+        { slug: "year-10", matched: "year 10" },
+        { slug: "year-11", matched: "year 11" },
+      ],
+      "year-10",
+    ],
```

Comment:
I personally find this quite hard to read. I need to jump from the test to the data to understand what each string represents. Given that the test code is so small, have you considered just having two tests? I don't even think it would be fewer lines

If this is a standard practice at Oak then it's totally fine and something I can adjust to

Reply:
Please ignore this for now. I can see how it makes context in the rest of the tests where it's a less clunky example